src/button/notification.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-button-notification($status: primary) {
+  position: relative;
+
+  &::before {
+    content: ' ';
+    position: absolute;
+    width: $iui-s;
+    height: $iui-s;
+    border-radius: 100%;
+    @include themed {
+      background-color: t(iui-color-foreground-#{$status});
+      border: 2px solid t(iui-color-background-1);
+    }
+  }
+
+  &.iui-urgent::before {
+    @media (prefers-reduced-motion: no-preference) {
+      animation: pulse 2s infinite;
+    }
+  }
+
+  @media (prefers-reduced-motion: no-preference) {
+    @keyframes pulse {
+      0% {
+        @include themed {
+          box-shadow: 0 0 0 0 rgba(t(iui-color-foreground-#{$status}-rgb), t(iui-opacity-1));
```

Comment:
The only issue remaining is no matter what status is applied the `.iui-urgent` class is always displaying in red.  I am unsure why.

Reply:
I think the issue is that all keyframes has the same name: "pulse" and one status styling is overriding another. Try renaming to `pulse#{$status}` and using that name in `animation`.

Reply:
Amazing, that fixed it!  Thank you!